src/commands/auth.ts

```diff
@@ -107,6 +112,13 @@ that configuration file swiftly.`)
       return
     }
 
+    if (system) {
+      this.mixCLIConfig = Config.switchConfiguration(this.mixCLIConfig, system)
```

Comment:
Use a `try` clause so you can catch the error and then use `this.error` to exit and have oclif format the error message like the other error message. You will have to create an `EINVALIDMIXSYSTEM` error in src/utils/errors.ts.